release/_releases_json.sh

```diff
@@ -1,8 +1,20 @@
 #!/bin/bash
 
-function regenerate_releases_json {
-	_process_product dxp
-	_process_product portal
+function generate_releases_json {
+	if [ "${1}" = "regenerate" ]
+	then
+		_process_product dxp
+		_process_product portal
+	else
+		_process_new_product
+
+		local exit_code=${?}
+	fi
+
+	if [ "${exit_code}" -eq "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}" ]
+	then
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
```

Comment:
Hi @natocesarrego I changed the exit code part to look like [this](https://github.com/Wykenz/liferay-docker/blob/LPD-28712-modify-the-generation-of-releases.json-in-release-gold.sh/release/_releases_json.sh#L110). The previous solution and the one that you suggested [here](https://github.com/natocesarrego/liferay-docker/pull/99#discussion_r1665513718) ran the `_process_new_product` function multiple times and that duplicated the json snippet.